### ISSUE TITLE
LinterPhp.Linter is deprecated.

### ISSUE THREAD

**Issue by manoj-apare** [AtomLinter/linter-php#41]
AtomLinter v0.X.Y API has been deprecated. Please refer to the Linter docs to update and the latest API: https://github.com/AtomLinter/Linter/wiki/Migrating-to-the-new-API
```
LinterPhp.Linter (/home/user/.atom/packages/linter/lib/linter.coffee:60:4)
new LinterPhp (/home/user/.atom/packages/linter-php/lib/linter-php.coffee:24:4)
Object.lint (/home/user/.atom/packages/linter/lib/legacy.coffee:61:21)
<unknown> (/home/user/.atom/packages/linter/lib/editor-linter.coffee:68:23)
<unknown> (/home/user/.atom/packages/linter/lib/editor-linter.coffee:67:24)
Set.forEach (native)
```

**Comment by renanmpimentel** [comment]
+1

**Comment by Aclopes** [comment]
:+1:

**Comment by days85** [comment]
:+1:

**Comment by juvevood** [comment]
+1

**Comment by fredcamps** [comment]
+1

**Comment by CoreySwish** [comment]
+1

**Comment by rjp2525** [comment]
+1

**Comment by markatia** [comment]
+1

**Comment by simonhudin** [comment]
+1

**Comment by makerlabs** [comment]
+1

**Comment by jacobgelling** [comment]
:+1:

**Comment by tmarquesini** [comment]
+1

**Comment by patrickburke** [comment]
:+1:

**Comment by ndaidong** [comment]
+1

**Comment by steelbrain** [comment]
Fixed by #48
